__tests__/smoke-plugins/run-smoke.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -eo pipefail # Fail on script errors
+
+cd __tests__/smoke-plugins
+echo
+echo "Directory content:"
+ls -a
+echo
+
+# Install plugin package
+npm i
+
+# Install CLI globally
+npm i redocly-cli.tgz -g
```

Comment:
Let's move it under package.json's dependencies.

Reply:
To reproduce the original issue this PR fixes, the CLI needs to be installed globally.